### PR TITLE
Feat/instruction-message

### PR DESCRIPTION
**Pull request by JumpLink**
* Moved welcome message to `MessageRenderer`
* Add instruction message method to `MessageRenderer`
* Print welcome and instruction message on /start command
* Add new `Renderer` type for the [Parse Mode Plugin](https://grammy.dev/plugins/parse-mode)
* Only show list and start commands on startup
* Removed auto generated translations
* Fallback to English translation if translation was not found
* Removed auto translations for quality reasons

### CODE REVIEW

File: telegram-bot/locales/de.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
         "enterText": "Bitte halten Sie Ihre Antwort kurz, idealerweise in einem Satz.",
         "enterTime": "Bitte geben Sie eine Zeit ein. (z.B. 20:45)",
         "enterUrl": "Bitte geben Sie eine Webadresse ein. (z.B. https://www.beispiel.de)",
+        "intro": "Hallo! Ich bin der {botName} Bot. Ich kann dir eine Reihe von offenen Callouts zeigen, an denen du hier auf Telegram teilnehmen kannst. Du kannst die folgenden Befehle verwenden, um dies zu tun, wähle sie einfach im Menü aus oder schreibe sie mir direkt:\n\n{comands}",
```
Comment (rlafuente) [inline telegram-bot/locales/de.json:35]: typo in "commands"

File: telegram-bot/services/communication.service.ts [hunk 2]
```diff
@@ -32,36 +32,51 @@ export class CommunicationService {
...
-      await ctx.replyWithMediaGroup([res.photo]);
-      if (res.keyboard) {
-        await ctx.reply("Please select an option", {
-          reply_markup: res.keyboard,
+  public async send(ctx: Context, render: Render) {
+    switch (render.type) {
+      case RenderType.PHOTO:
+        await ctx.replyWithMediaGroup([render.photo]);
+        if (render.keyboard) {
+          await ctx.reply("Please select an option", {
```
Comment (rlafuente) [inline telegram-bot/services/communication.service.ts:47]: Hardcoded string, maybe should be moved to the locales sheet?

File: telegram-bot/commands/show.command.ts [hunk 3]
```diff
@@ -10,13 +10,16 @@ import {
 } from "../renderer/index.ts";
 
 import { Command } from "../core/index.ts";
+import type { UserState } from "../types/user-state.ts";
 
 @Singleton()
 export class ShowCommand extends Command {
   key = "show";
   command = "show";
   description = `Shows you information about a specific callout`;
```
Comment (rlafuente) [inline telegram-bot/commands/show.command.ts:19]: I might be misunderstanding, but if the command descriptions are reused in the messages, they should be imported from the locales sheet, no?

File: telegram-bot/services/communication.service.ts (hunk 2 above)
Comment (JumpLink) [reply telegram-bot/services/communication.service.ts:47]: yes 👍

File: telegram-bot/commands/show.command.ts (hunk 3 above)
Comment (JumpLink) [reply telegram-bot/commands/show.command.ts:19]: Yes, you are right 👍

File: telegram-bot/locales/de.json (hunk 1 above)
Comment (JumpLink) [reply telegram-bot/locales/de.json:35]: thanks 👍

**Pull request merged**